docs/cudf/source/conf.py

```
@@ -625,14 +636,73 @@ def on_missing_reference(app, env, node, contnode):
+# case here and silence the resulting warnings so the docs build still
+# succeeds; the autodoc/xref content will simply be absent until the
+# environment is fixed.
+try:
+    import cudf_polars.experimental.rapidsmpf.collectives.common
+    import cudf_polars.experimental.rapidsmpf.frontend.core
+    import cudf_polars.experimental.rapidsmpf.frontend.dask
+    import cudf_polars.experimental.rapidsmpf.frontend.hardware_binding
+    import cudf_polars.experimental.rapidsmpf.frontend.options
+    import cudf_polars.experimental.rapidsmpf.frontend.ray
+    import cudf_polars.experimental.rapidsmpf.frontend.spmd
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)